tests/test_bonded.py

```diff
@@ -1,31 +1,37 @@
 import numpy as np
 import jax
-import jax.numpy as jnp
-from jax.config import config; config.update("jax_enable_x64", True)
+from jax.config import config
+
+config.update("jax_enable_x64", True)
 import functools
 
 from common import GradientTest
 from timemachine.lib import potentials
 from timemachine.potentials import bonded
 
+relative_tolerance_at_precision = {
+    np.float32: 2e-5,
+    np.float64: 1e-9
+}
 
-class TestBonded(GradientTest):
-
-    def test_centroid_restraint(self):
 
-        N = 10
+class TestBonded(GradientTest):
 
-        for precision, rtol in [(np.float64, 1e-9), (np.float32, 2e-5)]:
+    def test_centroid_restraint(self, n_particles=10, n_A=4, n_B=3):
+        """Randomly define subsets A and B of a larger collection of particles,
+        generate a centroid restraint between A and B, and then validate the resulting CentroidRestraint force"""
 
-            x_primal = self.get_random_coords(N, 3)
+        kb = 5.4  # TODO: is this constant important?
```

Comment:
Not important, might as well as be a random number.

Reply:
but just remember to seed a fixed seed and test a couple of combinations

Reply:
Thanks! I will move this into the test argument (with 5.4 as default value), for later randomization